Python/traceback.c

```diff
@@ -563,6 +597,9 @@ tb_displayline(PyTracebackObject* tb, PyObject *f, PyObject *filename, int linen
     }
     
 done:
+    if (source_line) {
+        Py_DECREF(source_line);
+    }
```

Comment:
Why not just xdecref?
```suggestion
    Py_XDECREF(source_line);
```